package.json

```diff
@@ -7,32 +7,36 @@
   "description": "machine-readable, always up-to-date GitHub Webhooks specifications",
   "main": "index.json",
   "devDependencies": {
+    "@apidevtools/json-schema-ref-parser": "^9.0.6",
     "@octokit/request": "^5.0.0",
     "ajv": "^7.0.0",
     "cheerio": "^1.0.0-rc.2",
     "execa": "^5.0.0",
     "fs-extra": "^9.0.0",
     "json-diff": "^0.5.3",
+    "prettier": "^2.0.5",
     "semantic-release": "^17.0.0",
     "turndown": "^7.0.0",
     "yargs": "^16.0.0"
   },
   "scripts": {
-    "build": "node bin/octokit-webhooks.js check --cached",
+    "build": "npm run build:schema && npm run build:webhooks",
+    "build:schema": "node bin/octokit-schema.js",
+    "build:webhooks": "node bin/octokit-webhooks.js check --cached",
```

Comment:
To update the `index.json` file, the command is actually `node bin/octokit-webhooks.js update`.

The current `build` command, only updates the cache of webhooks docs and is used by this GitHub workflow: https://github.com/octokit/webhooks/blob/9b8cac417e14b5ff5ba7c456b44d431db305fe1c/.github/workflows/update.yml

Reply:
```suggestion
    "build:webhooks": "node bin/octokit-webhooks.js update",
```